include/mpd/recv.h

```diff
@@ -46,6 +46,14 @@ struct mpd_connection;
 extern "C" {
 #endif
 
+/**
+ * Reads the binary data from the server.  Returns NULL on error.
+ *
+ * The caller must free the allocated memory.
```

Comment:
Free, how? If your answer is `free()`: keep in mind that `free()` doesn't work across library boundaries on Windows, because each DLL has its own heap.

Reply:
Now the caller must allocate the buffer.